app/models/post.rb

```diff
@@ -3,6 +3,10 @@ class Post < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  validates :title, presence: true, length: { maximum: 250 }
```

Comment:
- [ ] Hi @SergSkol, what do you think of kindly adding a test for the scenario where the title of the post must not exceed 250 characters?